lib/domain/pull_request.rb

```diff
@@ -1,13 +1,12 @@
 module Domain
   class PullRequest
-    attr_reader :application_name, :title, :open_since, :url, :version
+    attr_reader :application_name, :title, :open_since, :url
 
     def initialize(application_name:, title:, opened_at:, url:)
       @application_name = application_name
       @title = title
       @open_since = days_open(opened_at)
       @url = url
-      @version = version_from_title(title)
```

Comment:
Shame you lose the version info.

Reply:
Yeaahh - It wasn't too useful though as they were all updated to the same version, but in the case of multi-gem bumps there's no way to get that information outside of viewing the actual file changes for every PR